src/crawlee/storage_clients/_file_system/_storage_client.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+
+from typing_extensions import override
+
+from crawlee._utils.docs import docs_group
+from crawlee.configuration import Configuration
+from crawlee.storage_clients._base import StorageClient
+
+from ._dataset_client import FileSystemDatasetClient
+from ._key_value_store_client import FileSystemKeyValueStoreClient
+from ._request_queue_client import FileSystemRequestQueueClient
+
+
+@docs_group('Classes')
+class FileSystemStorageClient(StorageClient):
+    """File system implementation of the storage client.
```

Comment:
Please mention that this is not safe for concurrent access (from multiple crawler processes)